docs/source/configuration/index.rst

```diff
@@ -14,3 +14,103 @@ The AtlasDB configuration has two main blocks - keyValueService and leader. Plea
    cassandra_KVS_configuration
    leader_config
    logging
+
+There are three valid configuration modes, listed below.
+
+Leader block
+------------
+
+This configuration setup is required to use Cassandra KVS with more than one node.
+
+A minimal AtlasDB configuration for running against cassandra will look like the below.
+
+Importantly - your lock creator must be consistent across all nodes.
+If you do not provide a lock creator, it will default to the lexicographically first host in the leaders list.
+Therefore, without a lock creator, the leaders block should be exactly the same across all nodes.
+
+.. code-block:: yaml
+
+    atlasdb:
+      keyValueService:
+        type: cassandra
+        servers:
+          - cassandra:9160
+        poolSize: 20
+        keyspace: yourapp
+        credentials:
+          username: cassandra
+          password: cassandra
+        sslConfiguration:
+          trustStorePath: var/security/truststore.jks
+        replicationFactor: 1
+        mutationBatchCount: 10000
+        mutationBatchSizeBytes: 10000000
+        fetchBatchCount: 1000
+        safetyDisabled: false
+        autoRefreshNodes: false
+
+      leader:
+        # This should be at least half the number of nodes in your cluster
+        quorumSize: 2
+        learnerLogDir: var/data/paxosLogs
+        acceptorLogDir: var/data/paxosLogs
+        # This should be different for every node. If ssl is not enabled, then the host must be specified as http
```

Comment:
Nit: "This should be..." -> "localServer should be..."  Similar for lockCreator

(At least, I prefer it that way because the lack of newlines makes it slightly confusing whether to look above or below)